xplique/attributions/base.py

```diff
@@ -74,15 +74,27 @@ def explain(self,
                 inputs: Union[tf.data.Dataset, tf.Tensor, np.array],
                 targets: Optional[Union[tf.Tensor, np.array]] = None) -> tf.Tensor:
         """
-        Compute the explanations of the given samples.
+        Compute the explanations of the given inputs.
         Accept Tensor, numpy array or tf.data.Dataset (in that case targets is None)
 
         Parameters
         ----------
         inputs
-            Input samples to be explained.
+            Dataset, Tensor or Array. Input samples to be explained.
+            If Dataset, targets should not be provided (included in Dataset).
+            Expected shape among (N, W), (N, T, W), (N, W, H, C). More information in the
+            documentation
+            (<a href=
+            "https://deel-ai.github.io/xplique/api/attributions/api_attributions/">
+            API Description </a>).
         targets
-            One-hot encoded labels or regression target (e.g {+1, -1}), one for each sample.
+            Tensor or Array. One-hot encoding of the model's output from which an explanation
```

Comment:
I found the doc easier before ^^ but I suppose I am biased on this one haha

Reply:
@AntoninPoche your opinion?

Reply:
I think the most important thing is that the user does not mix targets with the ground truth or the model predictions. I think that a line about this may be needed.
Nevertheless, I prefer Lucas' doc, I think that {-1, 1} is misleading as without further information, -1 should not be put in the targets. (It is not trivial to interpret).

Reply:
Ok go for the new one then ;)